dashboard/app/models/level_source.rb

```diff
@@ -19,6 +19,7 @@
 
 # A specific solution attempt for a specific level
 class LevelSource < ActiveRecord::Base
+  DELETED_BY_THE_SYSTEM = 'deleted_by_the_system'
```

Comment:
See #15733, specifically [here](https://github.com/code-dot-org/code-dot-org/pull/15733#discussion_r121192821), for conversation about what this string should be (in a similar context).